modules/ROOT/pages/errors/gql-errors/index.adoc

```diff
@@ -684,991 +602,978 @@ Status description:: error: invalid transaction termination - inner transactions
 
-=== 52N17
+=== xref:errors/gql-errors/52N17.adoc[52N17]
 
 Status description:: error: procedure exception - quarantine change failed. Setting/removing the quarantine marker failed.
 
-=== 52N18
+=== xref:errors/gql-errors/52N18.adoc[52N18]
 
 Status description:: error: procedure exception - too many seeders. The number of seeding servers `{ $countSeeders }` is larger than the defined number of allocations `{ $countAllocs }`.
 
-=== 52N19
+=== xref:errors/gql-errors/52N19.adoc[52N19]
 
 Status description:: error: procedure exception - no such seeder. The specified seeding server with id `{ $server }` was not found. Verify that the spelling is correct.
 
-=== 52N22
+=== xref:errors/gql-errors/52N22.adoc[52N22]
 
 Status description:: error: procedure exception - invalid procedure argument. Invalid argument `{ $field }` for `{ $procParam }` on procedure `{ $proc }`. The expected format of `{ $procParam }` is `{ $procParamFmt }`.
 
-=== 52N23
+=== xref:errors/gql-errors/52N23.adoc[52N23]
 
 Status description:: error: procedure exception - non-reloadable namespace. The following namespaces are not reloadable: `{ $namespaceList }`
 
-=== 52N25
+=== xref:errors/gql-errors/52N25.adoc[52N25]
 
 Status description:: error: procedure exception - JMX error. JMX error while accessing `{ $param }`. See logs for more information.
 
-=== 52N29
+=== xref:errors/gql-errors/52N29.adoc[52N29]
 
 Status description:: error: procedure exception - outdated change identifier. Given ChangeIdentifier describes a transaction that occurred before any enrichment records exist.
 
-=== 52N30
+=== xref:errors/gql-errors/52N30.adoc[52N30]
 
 Status description:: error: procedure exception - future change identifier. Given ChangeIdentifier describes a transaction that hasn't yet occurred.
 
-=== 52N31
+=== xref:errors/gql-errors/52N31.adoc[52N31]
 
 Status description:: error: procedure exception - wrong database. Change identifier `{ $param }` does not belong to this database.
 
-=== 52N32
+=== xref:errors/gql-errors/52N32.adoc[52N32]
 
 Status description:: error: procedure exception - invalid sequence number. Change identifier `{ $param1 }` has an invalid sequence number `{ $param2 }`.
 
-=== 52N33
+=== xref:errors/gql-errors/52N33.adoc[52N33]
 
 Status description:: error: procedure exception - procedure invocation failed. Failed to invoke procedure/function `{ $sig }` caused by: `{ $msg }`.
 
-=== 52N34
+=== xref:errors/gql-errors/52N34.adoc[52N34]
 
 Status description:: error: procedure exception - procedure sandboxed. `{ $sig }` is unavailable because it is sandboxed. Sandboxing is controlled by the dbms.security.procedures.unrestricted setting. Only un-restrict procedures you can trust with access to database internals.
 
-=== 52N35
+=== xref:errors/gql-errors/52N35.adoc[52N35]
 
 Status description:: error: procedure exception - procedure compilation failed. Failed to compile procedure/function defined in `{ $procClass }`: `{ $msg }`
```

Comment:
```suggestion
Status description:: error: procedure exception - procedure compilation failed. Failed to compile procedure defined in `{ $procClass }`: `{ $msg }`
```

I changed this one when created 53N35, but missed to update in https://github.com/neo4j/docs-status-codes/pull/243